cypress/support/accessibility/a11y-utils.ts

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable jest/no-export */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+// @ts-nocheck
```

Comment:
Is the intention to remove the ts-ignores at some point?

Reply:
those files `cypress/e2e/accessibility/**` will be removed after the refactor